code/modules/mob/living/silicon/ai/decentralized/expansion_card_holder.dm

```diff
@@ -77,10 +77,14 @@ GLOBAL_LIST_EMPTY(expansion_card_holders)
 
 /obj/machinery/ai/expansion_card_holder/update_icon()
 	cut_overlays()
-	
+
 	if(!(stat & (BROKEN|NOPOWER|EMPED)))
 		var/mutable_appearance/on_overlay = mutable_appearance(icon, "[initial(icon_state)]_on")
 		add_overlay(on_overlay)
+		if(installed_cards.len)
+			var/mutable_appearance/cards = mutable_appearance(icon, "[initial(icon_state)]_[installed_cards.len]")
```

Comment:
actually, clamp the installed_cards.len to 2

Reply:
But in theory it can have more than 2 installed cards, since it has a variable for that. Either way, how do you want me to "clamp" it?

Reply:
Well don’t clamp the actual length of the list, clamp the sprite number to 2
Use CLAMP()

Reply:
clamped to max_cards for good measure

Reply:
actually lets not do that just do 2